src/diffusers/models/attention_processor.py

```diff
@@ -56,7 +56,8 @@ def __init__(
         bias=False,
         upcast_attention: bool = False,
         upcast_softmax: bool = False,
-        cross_attention_norm: bool = False,
+        cross_attention_norm: Optional[Union[bool, str]] = False,
```

Comment:
Ideally, this should just be an `Optional[str]` type now but this is backwards compatible. In the unet and unet blocks, I treat the type as `Optional[str]` and it's still pretty clean